modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -130,12 +138,12 @@ status:
   +
 If the value is set to `false`, the snapshot was created. However, the storage backend needs to perform additional tasks to make the snapshot usable so that it can be restored as a new volume. For example, Amazon Elastic Block Store data might be moved to a different, less expensive location, which can take several minutes.
 
-To verify that the volume snapshot was created automatically, run the following command:
-
+. To verify that the volume snapshot was created automatically, run the following command:
++
```

Comment:
This verification step is not clear enough for me.
Here, we want to check if the snapshot is created in the underlying storage system?
So, we need to check if the `volumesnapshotcontent` in the `volumesnapshot.status.boundVolumeSnapshotContentName` is created or not?

Reply:
> This verification step is not clear enough for me.
> Here, we want to check if the snapshot is created in the underlying storage system?
> So, we need to check if the `volumesnapshotcontent` in the `volumesnapshot.status.boundVolumeSnapshotContentName` is created or not?

@qinpingli That is my understanding, yes. I updated wording so it is hopefully more clear now.